### DISCUSSION TITLE
When will the aforementioned promise be fulfilled?

### DISCUSSION THREAD

**Discussion by YasseenAlhusammi** [flet-dev/flet#3650] [General]
When will the promise recorded in the last ten lines of the page be fulfilled: (https://flet.dev/blog/flet-packaging-update) :
"The new version of flet build will use a custom-made virtual pip index. This index will analyze dependencies, detect non-pure packages, and offer to pip mobile packages. For all other packages, it will fall back to PyPI."
I am waiting on the hottest coals:
To make it:
"flet build android" is able to include non-pure packages, when will this dream come true.
Are there any indications that this version will soon be downloaded, which is capable of including non-clean packages?
And when I am eagerly waiting for that.
Thank you!

**Comment by ndonkoHenri** [comment]
Its a WIP.

**Reply by YasseenAlhusammi** [reply]
Thank you for the response, it is a wonderful work that we are eagerly awaiting.
